### ISSUE TITLE
during review, clarify what the selection of junior coders means

### ISSUE THREAD

**Issue by jlcohoon** [cos-archives/archival-project#22]
Presently, a senior coder must select which junior coders to include in their review. This is not clear from the instructions given, however. I suggest changing the present layout (below)
![screen shot 2015-01-05 at 1 59 34 pm](https://cloud.githubusercontent.com/assets/4965832/5618225/ae879d7c-94e4-11e4-8f23-82dc92bc20fe.png)

to one with further elaboration, and a check box in lieu of a dropdown:
![screen shot 2015-01-05 at 2 09 22 pm](https://cloud.githubusercontent.com/assets/4965832/5618226/b13b59a0-94e4-11e4-856c-5fb0a917fa03.png)

**Comment by lyndsysimon** [comment]
I agree that there needs to be more explanation, but your proposed solution doesn't provide enough information.

For each study or test, the SC must choose the corresponding study or test from each junior coder's submissions. The studies and tests may be in a different order for each coder, and it's possible they may not even have the same number of each.
